ui-tests/src/test/java/io/syndesis/qe/hooks/ClassTransformerHook.java

```diff
@@ -71,6 +72,13 @@ private void transform() {
             }).installOnByteBuddyAgent();
     }
 
+    private void transform() {
+        ByteBuddyAgent.install();
```

Comment:
Any reason this is not also inside the if?

Reply:
There was -> it should only be called once, but I changed it to lazy load it for each transformer.